components/browser/icons/src/main/java/mozilla/components/browser/icons/pipeline/IconResourceComparator.kt

```diff
@@ -41,9 +41,11 @@ internal object IconResourceComparator : Comparator<IconRequest.Resource> {
     }
 }
 
-@Suppress("MagicNumber")
+@Suppress("MagicNumber", "ComplexMethod")
 private fun IconRequest.Resource.Type.rank(): Int {
     return when (this) {
+        IconRequest.Resource.Type.MANIFEST_MASKABLE_ICON -> 25
+        IconRequest.Resource.Type.MANIFEST_ICON -> 20
```

Comment:
This needs further discussion. I'm not convinced that we should use the manifest icon everywhere in the app UI. Is that a good choice?